app/client/templates/exhibitor/payment.html

```diff
@@ -0,0 +1,334 @@
+<template name="payment">
+  <div class="section">
+    <h3>{{_ "exhibitor-payment"}}</h3>
+  </div>
+  <div class="card">
+    <div class="card-content">
+      <p class="flow-text">{{_ "exhibitor-please_pay"}}</p>
+      <form id="pay" enctype="application/x-www- form-urlencoded" action="" method="post">
+        <input type="hidden" value="{{ _id }}" id="application-id">
+        <div class="row">
+          <div class="col s12 m6">
+            <label for="dollars" class="required">{{_ "exhibitor-pay-dollars"}}</label>
+            <input type="number" name="dollars" id="dollars" size="5" placeholder="100" required>
+          </div>
+          <div class="col s12 m6">
+            <label for="cents" class="required">{{_ "exhibitor-pay-cents"}}</label>
+            <input type="number" name="cents" id="cents" size="2" placeholder="00" required>
```

Comment:
`The size attribute specifies the visible width, in characters, of an <input> element.`